.gitignore

```diff
@@ -1,3 +1,7 @@
 target
 Cargo.lock
 /.cargo/config
+.project
+.settings
+*~
+*.bk
```

Comment:
Thanks. I removed that commit from the pull request (and also rebased the pull request).